flintrock/ec2.py

```diff
@@ -661,12 +725,12 @@ def _create_instances(
         block_device_mappings,
         availability_zone,
         placement_group,
-        tenancy,
         security_group_ids,
         subnet_id,
         instance_profile_arn,
-        ebs_optimized,
-        instance_initiated_shutdown_behavior,
+        tenancy='default',
```

Comment:
@dm-tran 
There is no default value to other parameters.

is it the right place to provide them ? Or should them be provided in a previous step ?